examples/js_of_ocaml/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# Compile OCaml source file to OCaml bytecode
+ocamlbuild -use-ocamlfind \
+  -tags "warn(-40)" \
+  -pkgs lwt.syntax,js_of_ocaml,js_of_ocaml.syntax,js_of_ocaml.tyxml,tyxml,js_of_ocaml.deriving,js_of_ocaml.deriving.syntax,deriving \
+  -syntax camlp4o \
+  todomvc.byte ;
+
+# Build JS code from the OCaml bytecode
+js_of_ocaml +weak.js --opt 3 -o js/todomvc.js todomvc.byte
```

Comment:
Can you add a description to the readme about how to bulid the app? That makes it clear for people and helps us to maintain the app further on.